.github/workflows/build_pyvelox.yml

```diff
@@ -46,6 +46,8 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  # TODO: Disable this job until we create a new docker image.
+  if: false
```

Comment:
Can we remove TODO:.. and instead point to this issue : https://github.com/facebookincubator/velox/issues/9014